src/components/Dropdown/Dropdown.react.js

```diff
@@ -170,8 +171,8 @@ class Dropdown extends React.Component<Props, State> {
     );
 
     const trigger = (() => {
-      if (this.props.trigger) return this.props.trigger;
-      if (this.props.icon || this.props.triggerContent) {
+      if (props.trigger) return props.trigger;
```

Comment:
[nit] could just destructure these one by one on line 160 instead of `...props` and then we won't have to do `props.trigger` etc we could just do `trigger`

Reply:
Yes most certainly!

Reply:
Oh now I remember!  Due to the prop types flow doesn't know those props exist yet at line 160. I also now realize that the other recent one I fixed was this file and it's over usage of this.props.